### PR TITLE
Issues/29

### PR DESCRIPTION
**Pull request by matmar10**
Demonstrates valid behavior present in master but failing in latest npm version (3.0.1) as describe idn issue #29

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1661,3 +1661,62 @@ test('original various tests', function (t) {
 
   t.end();
 });
+
+test('map array inside array to property', function (t) {
+
+  var mongoose = require('mongoose');
```
Comment (srogers202) [inline test/test.js]: Any reason for this?

Comment (therealklanni) [inline test/test.js]: Nope

**Comment by matmar10** [comment]
Sorry about that, had left a line in from my own local testing (making sure there wasn't some mongoose magic causing an issue. Removed the offending line and also formatted indenting of the new test case.

**Comment by matmar10** [comment]
Bump.

**Comment by therealklanni** [comment]
Paging @wankdanker :)

**Comment by wankdanker** [comment]
Sorry everyone. Merged and published to npm object-mapper@3.1.0.

**Pull request merged**